agents/getFiles.js

```diff
@@ -4,17 +4,22 @@ const { StructuredOutputParser, OutputFixingParser } = require('langchain/output
 const { getModel } = require('../modules/model');
 const { saveLog } = require('../modules/fsOutput');
 
-const promptTemplate = `USER INPUT: {task}
-YOUR TASK: 
-Identify the minimal set of existing files, from the provided context bellow, to read based on the USER INPUT. 
-Also explain why the file was selected.
+const promptTemplate = 
+`
+# User Input
+## This is what the user requested
+{task}
+
+# Your Role
+## This is your role
+Identify the files needed for the user input. Don't include new files. Also explain why the file was selected.
+Take into consideration that "user inputs" can be questions, code changes, reports of bugs or others. Reply accordingly.
 
 {format_instructions}
 
-CONTEXT:
-*** START REPOSITORY CONTEXT ***
+# CONTEXT
+## This is the context of the project
 {summaries}
-*** END REPOSITORY CONTEXT ***
 `;
```

Comment:
This agent now has unit tests https://github.com/fjrdomingues/autopilot/blob/main/agents/getFiles.test.js
Before changing the prompt, please add tests and check them before and after against both gpt3.5 and gpt4.